pkg/dao/mocks/resource.go

```diff
@@ -70,3 +74,7 @@ func (d *resourceDaoMock) FindBySource(ctx context.Context, source string) (api.
 func (d *resourceDaoMock) All(ctx context.Context) (api.ResourceList, error) {
 	return d.resources, nil
 }
+
+func (d *resourceDaoMock) AllBundle(ctx context.Context) (api.ResourceList, error) {
+	return api.ResourceList{}, nil
```

Comment:
ResourceList or ResourceBundleList?

Reply:
Do we really need a new type? Currently, all resources (single and bundle) are stored in the resources table in the database. To simplify, I only converted the resource bundle before writing the HTTP response.